CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- MongoDB HasMany - Allow documents embedded to an array to be editable.
```

Comment:
Users can edit a document in an array of embedded documents?

Reply:
Done